nbparameterise/code.py

```diff
@@ -35,10 +35,19 @@ def __eq__(self, other):
                 and self.type == other.type
                 and self.value == other.value
             )
+def findFirstParametersTaggedCell(nb):
+    for cell in nb.cells:
+        if 'metadata' in cell:
+            if 'tags' in cell['metadata']:
+                if any([i.lower()=="parameters" for i in cell['metadata']['tags']]):
```

Comment:
Oh, one more thought: rather than lots of `if` statements, it might be neater to do something like this:

```python
tags = cell.get('metadata', {}).get('tags', [])
```

Up to you, though. :slightly_smiling_face: